### DISCUSSION TITLE
cli: option to set description

### DISCUSSION THREAD

**Discussion by bkil** [asciinema/discussions#137] [Ideas]
Each playback page seems to include a `.description` div, probably editable after logging in with an account. Could we perhaps also fill this in before registering from the command line?

To be more precise, I am asking that every possible metadata that is visible on the webpage should also have a field in the JSON to allow for archiving or publishing to multiple platforms.

**Comment by ku1ik** [comment]
Since CLI v3.2.0 a description can be set with `asciinema upload --description "..." your.cast` - see https://github.com/asciinema/asciinema/releases/tag/v3.2.0
